packages/roosterjs-react-emoji/lib/components/EmojiPane.tsx

```diff
@@ -265,7 +268,7 @@ export default class EmojiPane extends React.PureComponent<InternalEmojiPaneProp
                     </div>
                 </div>
 
-                <EmojiStatusBar emoji={this.getSelectedEmoji()} strings={strings} />
+                <EmojiStatusBar emoji={this.getSelectedEmoji()} strings={strings} {...statusBarProps} />
```

Comment:
let's move this before strings prop like navbar

Reply:
navbar has strings before the navbarprops though. should strings come before?